src/js/components/tasks/cards/orgActions.tsx

```diff
@@ -0,0 +1,62 @@
+import Button from '@salesforce/design-system-react/components/button';
+import Dropdown from '@salesforce/design-system-react/components/menu-dropdown';
+import i18n from 'i18next';
+import React from 'react';
+
+import { LabelWithSpinner } from '@/components/utils';
+import { Org } from '@/store/orgs/reducer';
+
+const OrgActions = ({
+  org,
+  ownedByCurrentUser,
+  assignedToCurrentUser,
+  ownedByWrongUser,
+  isCreating,
+  isDeleting,
+  doCreateOrg,
+  doDeleteOrg,
+}: {
+  org: Org | null;
+  ownedByCurrentUser: boolean;
+  assignedToCurrentUser: boolean;
+  ownedByWrongUser: Org | null;
+  isCreating: boolean;
+  isDeleting: boolean;
+  doCreateOrg: () => void;
+  doDeleteOrg: () => void;
+}) => {
+  if (isCreating) {
+    return (
+      <Button
+        label={<LabelWithSpinner label={i18n.t('Creating Org…')} />}
+        disabled
+      />
+    );
+  }
+
+  if (!isDeleting && (ownedByWrongUser || (org && ownedByCurrentUser))) {
+    return (
+      <Dropdown
+        align="right"
+        assistiveText={{ icon: i18n.t('Org Actions') }}
+        buttonClassName="slds-button_icon-x-small"
+        buttonVariant="icon"
+        iconCategory="utility"
+        iconName="down"
+        iconSize="small"
+        iconVariant="border-filled"
+        width="xx-small"
+        options={[{ id: 0, label: i18n.t('Delete Org') }]}
```

Comment:
maybe this is a react thing we can't avoid, and I'm not sure if this comment is in the right place, but "Delete Org" is the link text and the link title attribute which is redundant for screen readers if the title is exactly the same or doesn't provide additional useful info.

Reply:
Yeah, that's SLDS and matches the markup in SLDS, not something we control: https://lightningdesignsystem.com/components/menus/#Base